<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EARComm</class>
 <widget class="QWidget" name="EARComm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EARComm</string>
  </property>
  <widget class="QPushButton" name="readButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Read</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="programStatus">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>240</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="programButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>270</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Program</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="eventCodes">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>10</y>
     <width>421</width>
     <height>191</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Code</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Event Description</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="x10House">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>210</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="x10Unit">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>210</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="FIPS1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="FIPS2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="FIPS3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="FIPS4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="priFreq">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="altFreq">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="eomSwitch">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset on end-of-message</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="backupSwitch">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Automatically switch frequency</string>
   </property>
  </widget>
  <widget class="QComboBox" name="FIPS1St">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="FIPS2St">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="FIPS3St">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="FIPS4St">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="testButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>270</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelX10">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X-10 House/Unit:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrimary">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrimary_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Alternate:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
